client/dashboard/sites/settings-wpcom-login/summary.tsx

```diff
@@ -0,0 +1,42 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useQuery } from '@tanstack/react-query';
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { key } from '@wordpress/icons';
+import { siteJetpackModulesQuery } from '../../app/queries/site-jetpack-module';
+import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import { HostingFeatures, JetpackModules } from '../../data/constants';
+import { hasHostingFeature } from '../../utils/site-features';
+import type { Site } from '../../data/types';
+import type { Density } from '@automattic/components/src/summary-button/types';
+
+export default function WpcomLoginSettingsSummary( {
+	site,
+	density,
+}: {
+	site: Site;
+	density?: Density;
+} ) {
+	const { data: jetpackModules } = useQuery( siteJetpackModulesQuery( site.ID ) );
+
+	if ( ! isEnabled( 'dashboard/v2/security-settings' ) ) {
+		return null;
+	}
+
+	const ssoEnabled = jetpackModules?.includes( JetpackModules.SSO ) ?? false;
+
+	const badges =
+		ssoEnabled || ! hasHostingFeature( site, HostingFeatures.SECURITY_SETTINGS )
+			? [ { text: __( 'Enabled' ), intent: 'success' as const } ]
```

Comment:
On Simple sites, the summary will say `Enabled` but when I click, I get a callout. I think that's confusing? Should we even upsell this feature? Or we could just not show badge at all for such sites.

Reply:
I think this is a similar case than the PHP version. On simple sites, we display a badge with the current PHP version and if they click on it, we upsell the feature in case they are interested in changing it.

Here, we upsell the SSO feature for those interested in controlling the authentication needed to access their sites. 

Maybe we can change the description from "Allow registered users to log in to your site with their WordPress.com accounts" to something else that implies that the setting can be turned off too (e.g. "Manage logins to your site using WordPress.com’s secure authentication").

Reply:
I guess we should not the badge if the site doesn't have the feature 🤔

Reply:
🤔 I guess you're making a good point.

My reasoning is that WPCOM SSO is inherently enabled on Simple, where it's the only login option.
However, yeah, the "enabled" badge is confusing, as it suggests you can toggle it.
Omitting the badge entirely on Simple would probably be clearer.